File: CS/BlazorServerApp/Services/SupportedCultures.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Threading.Tasks;

namespace BlazorServerApp.Services {
    public class SupportedCultures {
        public static CultureInfo[] Cultures { get; } = new CultureInfo[] {
            new CultureInfo("de"),
            new CultureInfo("en"),
            new CultureInfo("es"),
            new CultureInfo("ja"),
            new CultureInfo("ru"),
        };
    }
}

File: VB/BlazorServerApp/Services/SupportedCultures.vb
Imports System.Globalization

Namespace BlazorServerApp.Services

    Public Class SupportedCultures

        Public Shared ReadOnly Property Cultures As CultureInfo() = New CultureInfo() {New CultureInfo("de-DE"), New CultureInfo("en-US"), New CultureInfo("ru-RU")}
    End Class
End Namespace
